tests/attributes/src/lib.rs

```diff
@@ -124,13 +124,53 @@ mod newtype_pattern {
     }
 }
 
+#[hax::fstar_before(r#"let before_${inlined_code} = "example before""#)]
```

Comment:
We should namespace this instead of having longer macro names.
How about something like `hax::fstar::before`?

Reply:
Yes, I was thinking about that as well, but abandoned the idea because we cannot export anything else than proc macros inside a proc-macro crate :(
But now you bring that up again, I could just re-export those long names in `hax-lib`, wdyt?

Reply:
Ah right... re-exporting would certainly be good to ensure that only one import is needed. Feature guarded so that it doesn't need to be consumed.